public/theme/boltcms-2021/featureslist.twig

```diff
@@ -0,0 +1,108 @@
+                <span>developers</span>
+                <span>designers</span>
+                <span>content creators</span>
+                <span>you</span>
+            </div>
+        </h1>
+        <div class="button-row">
+            {% include "partials/_features_buttons.twig" %}
+        </div>
+    </div>
+    {# {% set current = app.request.requestUri|split('/')|last %} #}
+
+    {# {% if current == 'features-developers' %} #}
+    {# {% setcontent developers = 'features' where { audience: 'developers' } order "order" %} #}
+    {# {% elseif current == 'features-designers' %} #}
+    {# {% setcontent features = 'features' where { audience: 'designers' } order "order" %} #}
+    {# {% elseif current == 'features-editors' %} #}
+    {# {% setcontent features = 'features' where { audience: 'editors' } order "order" %} #}
+    {# {% else %} #}
+    {# Assume we want to show all #}
+    {# {% setcontent features = 'features' where { audience: 'overview' } order "order" %} #}
+    {# {% endif %} #}
+
+    {# {{ record.content }} #}
+
+    {# {% include 'partials/_bolt_blockquote.twig' with { 'blockquote' : blockquote} %} #}
+
+    {# {% setcontent developers = 'features' where { audience: 'developers' } latest limit 2 %} #}
+
+    {# {% for developer in developers %} #}
+    {# <h3 class="titel2 text-left">{{ developer.title }}</h3> #}
+    {# {{ developer.content }} #}
+    {# {% endfor %} #}
+
+    {# {% for feature in features %} #}
+    {# <div class="row flex align-items-center my-5 {% if loop.index % 2 == 0 %}flex-row-reverse{% endif %}"> #}
+    {# {% if feature.image is not empty %} #}
+    {# <div class="col-md p-4 zoom-images"> #}
+    {# <a href="{{ feature.image|thumbnail(1600, 1600) }}"/> #}
+    {# <img class="img-fluid" src="{{ feature.image|thumbnail(600, 400) }}" loading="lazy" #}
+    {# alt="{{ feature.image.alt }}"> #}
+    {# </a> #}
+    {# </div> #}
+    {# {% endif %} #}
+    {# <div class="col-md p-4"> #}
+    {# <h3 class="titel2 text-left">{{ feature.title }}</h3> #}
+    {# {{ feature.content }} #}
+    {# {% if feature|edit_link() %}<a href="{{ feature|edit_link() }}" class="btn btn-info btn-sm">edit #}
+    {# <small>{{ feature.order|default('-') }}</small></a>{% endif %} #}
+    {# </div> #}
+    {# </div> #}
+    {# {% endfor %} #}
+
+    {# <div class="row my-5"> #}
+    {# {% include "partials/_features_buttons.twig" %} #}
+    {# </div> #}
+
+    {# {% include 'partials/_cta.twig' %} #}
+    {# <script> #}
+    {# window.addEventListener('load', function () { #}
+    {# baguetteBox.run('.zoom-images'); #}
+    {# }); #}
+    {# </script> #}
+    <div class="content-wrapper">
+        <div class="content-block">
+            <a id="developer"></a>
+            <div class="title">
+                <h2>For developers</h2>
+            </div>
+            <div class="features">
+                {% setcontent developers = 'features' where { audience: 'developers' } limit 2 %}
```

Comment:
A couple of things:

- If we limit the features to `2`, we can't show all features for developers/designers/content creators that we have content for. I don't think we should do that, let's show them all! In case there's no design for it, we may need to ask Derk to help adjust it.
- If we have the `setcontent` like this, we cannot reuse the `featureslist.twig` template for the specific pages, where we show features _only_ for developers, _only_ for designers, and _only_ for content creators. Here is an example of how it works right now:
https://github.com/bolt/boltcms.io/blob/bce7120616cf4d5b1aa0cc7c822b777d53b9428f/public/theme/boltcms-2020/featureslist.twig#L8-L17

- Last thing, let's keep the `order: "order"` part of the `setcontent` tag, that way we can adjust the order from the Editor :-)

Reply:
Check

Reply:
Check. 
Ps. In the overview I won't recommend to show them all. It's getting way too long and busy. But that's for discussion.

Reply:
I re-code it and it work now like it should do.